app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def new
+
+  end
```

Comment:
I'm guessing you're implementing this in your project, but encourage you to implement new/create functionality here if you need more practice with it.